Deprecate SHA1 based algorithms for SAML signatures (#1739)
keycloak/keycloak-documentation: server_admin/topics/clients/saml/proc-creating-saml-client.adoc

*Sign Assertions*:: The assertion is signed and embedded in the SAML XML Auth response.

> *Signature Algorithm*:: The algorithm used in signing SAML documents. Note that `SHA1` based algorithms are deprecated and can be removed in the future.

Reviewer: ```suggestion
*Signature Algorithm*:: The algorithm used in signing SAML documents. Note that `SHA1` based algorithms are deprecated and may be removed in a future release.
```

> It is recommended to stick to some more secure algorithm instead of `*_SHA1`. Also with `*_SHA1` algorithms verifying signatures
> will not work if the SAML client runs on Java 17 or later.

Reviewer: ```suggestion
We recommend the use of some more secure algorithm instead of `*_SHA1`. Also, with `*_SHA1` algorithms, verifying signatures
do not work if the SAML client runs on Java 17 or higher.
```
